Resources/views/CRUD/edit_orm_one_to_many.html.twig

```diff
@@ -120,10 +144,20 @@ file that was distributed with this source code.
 
         {% if sonata_admin.edit == 'inline' %}
 
-            {% if sonata_admin.field_description.associationadmin.hasroute('create') and sonata_admin.field_description.associationadmin.isGranted('CREATE') and btn_add and ( sonata_admin.field_description.options.limit is not defined or form.children|length < sonata_admin.field_description.options.limit ) %}
+            {% if sonata_admin.field_description.associationadmin.hasroute('create')
+                and sonata_admin.field_description.associationadmin.isGranted('CREATE')
+                and btn_add
+                and (
+                    sonata_admin.field_description.options.limit is not defined or
+                    form.children|length < sonata_admin.field_description.options.limit
+                )
+            %}
```

Comment:
Doesn't the perfectly aligned `%}` that you can actually see without scrolling, as opposed to assuming its presence, make the location of the tag end quicker to figure out? 
Also, isn't it great to see the structure and contents of this big condition at once, especially the part with the parenthesis?